.github/workflows/build-images-ci.yaml

```
@@ -244,6 +244,30 @@ jobs:
             NOSTRIP=1
             OPERATOR_VARIANT=${{ matrix.name }}
 
+      - name: Install Bom and generate SBOM
+        shell: bash
+        run: | 
+          curl -L https://github.com/kubernetes-sigs/bom/releases/download/v0.3.0/bom-linux-amd64.tar.gz -o bom 
+          tar -xvf bom
+          ./bom generate -o sbom_source_${{ matrix.name }}_${{ steps.tag.outputs.tag }}.spdx \
```

[Nit 3/3] ...so that we can use `bom` without prefix since it's in the path.
```suggestion
          bom generate -o sbom_source_${{ matrix.name }}_${{ steps.tag.outputs.tag }}.spdx \
```